liquidcore/config/agent.py

```diff
@@ -0,0 +1,328 @@
+"""
+Configuration agent for liquid-core
+
+This module does daemonization voodoo. It has two public methods: `launch` and
+`status`.
+
+## `agent.launch(target_configuration, repair)`
+`target_configuration` is the configuration that will be applied.
+
+`repair` - if the last job failed, should we attempt to recover? This flag
```

Comment:
Maybe explain what it does. Reruns the last working configuration? Reruns the last broken configuration? Unmarks failures and releases the flock? Mines bitcoin for 2 minutes?